extensions/amp-analytics/0.1/config.js

```diff
@@ -249,6 +251,26 @@ export class AnalyticsConfig {
     });
   }
 
+  /**
+   * Checks if config has opted into persisting any
+   * EventTimestamps the session. Sets the PERSIST_EVENT_TIMESTAMP
+   * flag in the config if so.
+   * @private
+   */
+  checkPersistEvents_() {
+    if (!this.config_['triggers']) {
+      return;
+    }
+    const keys = Object.keys(this.config_['triggers']);
+    for (let i = 0; i < keys.length; i++) {
+      const key = keys[i];
+      if (this.config_['triggers']?.[key]?.['session']?.['persistEvent']) {
+        this.config_[PERSIST_EVENT_TIMESTAMP] = true;
+        break;
+      }
+    }
```

Comment:
```suggestion
    this.config_[PERSIST_EVENT_TIMESTAMP] = Object.values(
      this.config_['triggers']
    ).some(trigger => trigger?.['session']?.['persistentEvent']);
```

Reply:
At minimum, at least iterate through the values directly via `Object.values(...)` rather than re-indexing over each of the keys

Reply:
Ignore above, I saw the comments resolved but not the changes before.